### DISCUSSION TITLE
Extension Manager not loading

### DISCUSSION THREAD

**Discussion by t-vanasseldonk** [phcode-dev/phoenix#1941] [Q&A]
I just installed Phoenix code on my Mac (M1 Pro, MacOS 14.6.1) for the first time, and I wanted to install some extensions. But opening the Extension Manager only shows the loading animation and doesn't seem to do anything else. 

The console shows these error messages a few seconds after opening the Extension Manager (some redundant info left out, can post the full output if it helps):

`[Error] Caught Critical error, Reported: 
_handledState: {severity: "error", unhandled: true, severityReason: {type: "unhandledException"}}
app: {releaseStage: "tauri-production", version: "3.9.4-20562", type: "tauri", duration: 8326}
context: "supported-tauri-desktop-mac-webkit"
device: {locale: "en-GB", userAgent: "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko)", orientation: "landscape-primary", id: "########", time: Thu Nov 21 2024 14:23:45 GMT+0100 (Central European Standard Time)}
errors:  {
    errorClass: "QuotaExceededError"
    errorMessage: "The quota has been exceeded."
    stacktrace: [
          {file: "[native code]", method: "setItem", lineNumber: 1, columnNumber: 451297, code: undefined, …},
          {file: "phtauri://localhost/v3.9.4/brackets.js", method: undefined, lineNumber: 1, columnNumber: 451297, code: undefined, …},
          {file: "phtauri://localhost/v3.9.4/thirdparty/jquery-2.1.3.min.js", method: "j", lineNumber: 1, columnNumber: 26896, code: undefined, …},
          {file: "phtauri://localhost/v3.9.4/thirdparty/jquery-2.1.3.min.js", method: "fireWith", lineNumber: 1, columnNumber: 27699, code: undefined, …},
          {file: "phtauri://localhost/v3.9.4/thirdparty/jquery-2.1.3.min.js", method: "x", lineNumber: 1, columnNumber: 75044, code: undefined, …},
          {file: "phtauri://localhost/v3.9.4/thirdparty/jquery-2.1.3.min.js", method: undefined, lineNumber: 1, columnNumber: 78543, code: undefined, …}
    ]
}
type: "browserjs"
groupingHash: undefined
originalError: QuotaExceededError: The quota has been exceeded.
request: {clientIp: "[REDACTED]", url: "phtauri://localhost/v3.9.4/"}
severity: "error"
unhandled: true`

`QuotaExceededError: The quota has been exceeded.`

I tried searching here for similar issues but couldn't find anything, so I assume this is not a know issue in Phoenix Code Version 3.9.4, but rather something unique to my installation? I've already tried re-installing, but that didn't make a difference. Anything else I should try to troubleshoot?

**Comment by abose** [comment]
Thanks for reporting the issue @t-vanasseldonk 
This has been fixed today as we got a few reports from users. see https://github.com/phcode-dev/phoenix/pull/1937
Not sure what changed suddenly with the quota limits in macs, seems recent as we got 3 reports today. Maybe some MacOS updates?

Fix will be available in the next release scheduled for mid December. But we may release it sooner if this is a wider issue.
